app/code/Transoft/Blog/Block/Adminhtml/Post/Edit/BackButton.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Block\Adminhtml\Post\Edit;
+
+use Magento\Cms\Block\Adminhtml\Block\Edit\GenericButton;
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+
+/**
+ * Class BackButton
```

Comment:
Please do not use the description that repeats the class name. Please provide meaningful class description. Fix in **all** files

Reply:
fixed